README.md

```diff
@@ -0,0 +1,51 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project 1: Positioning and Floating Elements
+
+> Learning how to position and float elements by cloning an article on New York Times
+
+
+## Built With
+
+- Languages - HTML, CSS
+- Frameworks - boostraps
+- Technology - Linters
+
+To get a local copy up and running follow these simple example steps.
+
+### Go to github.com/francisuloko/new-york-times-clone
+
+### Click Code button
+
+### Open Terminal on your machine
+
+### Enter git clone <repo path>
+
+## Authors
+
+👤 **FRancic Uloko**
+
+- GitHub: [@githubhandle](https://github.com/francisuloko]
+- Twitter: [@twitterhandle](https://twitter.com/FrancisUloko]
+- LinkedIn: [LinkedIn](https://linkedin.com/in/FrancisUloko]
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
add a link to your partner's profile: https://github.com/mrramoun

Reply:
Updated. Thanks.